### PR TITLE
FLUT-873260 - [documentation] Added How to section to widgets

### PR DESCRIPTION
**Pull request by AswiniDileep**
## Description ##

Except Maps added How to section page to all widgets as mentioned below:
Images also need to add properly.

Radial Gauge:
- Created a "How to" markdown file for Radial Gauge.
- The markdown file contains step-by-step instructions on adding the Radial Gauge widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Radial Gauge in the Flutter Flow framework.

Linear Gauge:
- Created a "How to" markdown file for Linear Gauge.
- The markdown file contains step-by-step instructions on adding the Linear Gauge widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Linear Gauge in the Flutter Flow framework.

Barcodes:
- Created a "How to" markdown file for Barcodes.
- The markdown file contains step-by-step instructions on adding the Barcodes widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Barcodes in the Flutter Flow framework.

Slider:
- Created a "How to" markdown file for Slider.
- The markdown file contains step-by-step instructions on adding the Slider widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Slider in the Flutter Flow framework.

Range Slider:
- Created a "How to" markdown file for Range Slider.
- The markdown file contains step-by-step instructions on adding the Range Slider widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Range Slider in the Flutter Flow framework.

Spark Chart:
- Created a "How to" markdown file for Spark Chart.
- The markdown file contains step-by-step instructions on adding the Spark Chart widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Spark Chart in the Flutter Flow framework.

Treemap:
- Created a "How to" markdown file for Treemap.
- The markdown file contains step-by-step instructions on adding the Treemap widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Treemap in the Flutter Flow framework.

Calendar:
- Created a "How to" markdown file for Calendar.
- The markdown file contains step-by-step instructions on adding the Calendar widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Calendar in the Flutter Flow framework.

Date Range Picker:
- Created a "How to" markdown file for Date Range Picker.
- The markdown file contains step-by-step instructions on adding the Date Range Picker widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Date Range Picker in the Flutter Flow framework.

SignaturePad:
- Created a "How to" markdown file for SignaturePad.
- The markdown file contains step-by-step instructions on adding the SignaturePad widget in the Flutter Flow framework.
- Included images illustrating the process of adding the SignaturePad in the Flutter Flow framework.

Cartesian Chart:
- Created a "How to" markdown file for Cartesian Chart.
- The markdown file contains step-by-step instructions on adding the Cartesian Chart widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Cartesian Chart in the Flutter Flow framework.

Funnel Chart:
- Created a "How to" markdown file for Funnel Chart.
- The markdown file contains step-by-step instructions on adding the Funnel Chart widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Funnel Chart in the Flutter Flow framework.

Pyramid Chart:
- Created a "How to" markdown file for Pyramid Chart.
- The markdown file contains step-by-step instructions on adding the Pyramid Chart widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Pyramid Chart in the Flutter Flow framework.

Circular Chart:
- Created a "How to" markdown file for Circular Chart.
- The markdown file contains step-by-step instructions on adding the Circular Chart widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Circular Chart in the Flutter Flow framework.

Range Selector:
- Created a "How to" markdown file for Range Selector.
- The markdown file contains step-by-step instructions on adding the Range Selector widget in the Flutter Flow framework.
- Included images illustrating the process of adding the Range Selector in the Flutter Flow framework.

### CODE REVIEW

**Review by VijayakumarMariappan — APPROVED** [review]
Fine

File: Flutter/treemap/how-to/custom-widget-on-flutterflow.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+![Compile code](how-to-section-images/compile-code.png)
+
+>**Note**: The compilation progress takes 2 to 3 minutes to complete.
+
+### Utilizing the custom widget
+
+1. Navigate to `Widget Palette` located in the left side navigation menu.
+2. Click on the `Componenets` tab.
+3. Under `Custom Code Widgets`, your custom widget will be there. Now, the custom widget can be added to your page by drag-and-drop.
```
Comment (M-A32) [inline Flutter/treemap/how-to/custom-widget-on-flutterflow.md:78]: 3. Your custom widget will be under `Custom Code Widgets`. Drag and drop the custom widget to your page.

File: Flutter/treemap/how-to/custom-widget-on-flutterflow.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+![Custom Widget](how-to-section-images/custom-widget.png)
+
+### Add Treemap widget as a dependency
+
+1. Click on `+ Add Dependency`, a text editor will appear.
+2. Navigate to [Syncfusion Flutter Treemap](https://pub.dev/packages/syncfusion_flutter_treemap) in [pub.dev](https://pub.dev/) and copy the dependency name and version using the `Copy to Clipboard` option.
+![Version](how-to-section-images/copy-version.png)
+3. Paste the copied dependency into the text editor, then click `Refresh` and `Save` it.
+
+>**Note**: The live version of [Syncfusion Flutter Treemap](https://pub.dev/packages/syncfusion_flutter_treemap) has been migrated to the latest version of Flutter SDK. To ensure compatibility, please check [FlutterFlow](https://app.flutterflow.io/dashboard)'s current Flutter version and obtain the corresponding version of [Syncfusion Flutter Treemap](https://pub.dev/packages/syncfusion_flutter_treemap) by referring to the [SDK compatibility](https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility).
```
Comment (M-A32) [inline Flutter/treemap/how-to/custom-widget-on-flutterflow.md:38]: >**Note**: The live version of [Syncfusion Flutter Treemap](https://pub.dev/packages/syncfusion_flutter_treemap) has been migrated to the latest version of Flutter SDK. To ensure compatibility, check [FlutterFlow](https://app.flutterflow.io/dashboard)'s current Flutter version and obtain the corresponding version of [Syncfusion Flutter Treemap](https://pub.dev/packages/syncfusion_flutter_treemap) by referring to the [SDK compatibility](https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility).

File: Flutter/treemap/how-to/custom-widget-on-flutterflow.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+[FlutterFlow](https://app.flutterflow.io/dashboard) enables you to create native applications using its graphical interface, reducing the need to write extensive amounts of code. Additionally, it offers the capability to include custom widgets that are not included in the default [FlutterFlow](https://app.flutterflow.io/dashboard) widget collection. This article explains how to incorporate our SfTreemap widget as a custom widget in [FlutterFlow](https://app.flutterflow.io/dashboard).
+
+### Create a new project
+
+Navigate to the [FlutterFlow dashboard](https://app.flutterflow.io/dashboard) and click the `+ Create New` button to create a new project.
+
+### Creating the custom widget
+
+1. Navigate to the `Custom Code` section in the left side navigation menu.
+2. Click on the `+ Add` button to open a dropdown menu. From there, select `Widget`.
```
Comment (M-A32) [inline Flutter/treemap/how-to/custom-widget-on-flutterflow.md:23]: 2. Click the `+ Add` button to open a dropdown menu. From there, select `Widget`.

File: Flutter/treemap/how-to/custom-widget-on-flutterflow.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Treemap widget in FlutterFlow | Syncfusion
```
Comment (M-A32) [inline Flutter/treemap/how-to/custom-widget-on-flutterflow.md:3]: title: Treemap Widget in FlutterFlow | Syncfusion

File: Flutter/sparkcharts/how-to/custom-widget-on-flutterflow.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+![Import](how-to-section-images/import-package-flutterflow.png)
+
+### Add widget code snippet in code editor
+
+1. Navigate to the [Example](https://pub.dev/packages/syncfusion_flutter_charts/example) tab in [Syncfusion Flutter Charts](https://pub.dev/packages/syncfusion_flutter_charts) and copy the widget specific codes.
+
+![Code](how-to-section-images/code-snippet.png)
+
+3. Paste the copied code snippet into the code editor, click `Format Code`, and `Save` it.
```
Comment (M-A32) [inline Flutter/sparkcharts/how-to/custom-widget-on-flutterflow.md:61]: 3. Paste the copied code sample into the code editor, click `Format Code`, and `Save` it.

File: Flutter/sparkcharts/how-to/custom-widget-on-flutterflow.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Spark Chart widget in FlutterFlow | Sparkline | Syncfusion
```
Comment (M-A32) [inline Flutter/sparkcharts/how-to/custom-widget-on-flutterflow.md:3]: title: Spark Chart Widget in FlutterFlow | Sparkline | Syncfusion

File: Flutter/slider/how-to/custom-widget-on-flutterflow.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+![Compile code](how-to-section-images/compile-code.png)
+
+>**Note**: The compilation progress takes 2 to 3 minutes to complete.
+
+### Utilizing the custom widget
+
+1. Navigate to `Widget Palette` located in the left side navigation menu.
+2. Click on the `Componenets` tab.
+3. Under `Custom Code Widgets`, your custom widget will be there. Now, the custom widget can be added to your page by drag-and-drop.
```
Comment (M-A32) [inline Flutter/slider/how-to/custom-widget-on-flutterflow.md:78]: 3. Your custom widget will be under `Custom Code Widgets`. Drag and drop the custom widget to your page.

File: Flutter/slider/how-to/custom-widget-on-flutterflow.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Slider widget in FlutterFlow | Syncfusion
```
Comment (M-A32) [inline Flutter/slider/how-to/custom-widget-on-flutterflow.md:3]: title: Slider Widget in FlutterFlow | Syncfusion

File: Flutter/signaturepad/how-to/custom-widget-on-flutterflow.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+
+![Import](how-to-section-images/import-package-flutterflow.png)
+
+### Add widget code snippet in code editor
+
+1. Navigate to the [Example](https://pub.dev/packages/syncfusion_flutter_signaturepad/example) tab in [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) and copy the widget specific codes.
+
+![Code](how-to-section-images/code-snippet.png)
+
+3. Paste the copied code snippet into the code editor, click `Format Code`, and `Save` it.
```
Comment (M-A32) [inline Flutter/signaturepad/how-to/custom-widget-on-flutterflow.md:61]: 3. Paste the copied code sample into the code editor, click `Format Code`, and `Save` it.

File: Flutter/signaturepad/how-to/custom-widget-on-flutterflow.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: SignaturePad widget in FlutterFlow | Syncfusion
```
Comment (M-A32) [inline Flutter/signaturepad/how-to/custom-widget-on-flutterflow.md:3]: title: SignaturePad Widget in FlutterFlow | Syncfusion

**Review by M-A32 — COMMENTED** [review]
I have completed the review.

**Pull request merged**